config/recipes/beats/filebeat_autodiscover_by_metadata.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: beat.k8s.elastic.co/v1beta1
```

Comment:
I feel like one or two sentences explaining what each config does would be helpful, especially for those of us who are not experts in the details of Beats configuration 

Maybe either have a subdirectory with README for each or alternatively a shared README with short explanation for each wdyt?